biojava-structure/src/test/java/org/biojava/nbio/structure/TestKeywords.java

```diff
@@ -0,0 +1,41 @@
+package org.biojava.nbio.structure;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
+import org.biojava.nbio.structure.io.PDBFileParser;
+import org.junit.Test;
+
+public class TestKeywords {
+
+	@Test
+	public void testKeywordsOnFiveLines () throws IOException {
+		String fileName = "/3cdl.pdb";
+		InputStream inStream = this.getClass().getResourceAsStream(fileName);
+
+		PDBFileParser pdbpars = new PDBFileParser();
+		Structure structure = pdbpars.parsePDBFile(inStream);
+		List<String> keywords = structure.getKeywords();
+		assertEquals(keywords.size(), 12);
+		assertEquals(keywords.get(11), "TRANSCRIPTION REGULATOR");
+	}
+
+	@Test
+	public void testDash() throws IOException {
+		String fileName;
+		fileName = "/pdb6elw.ent.gz";
```

Comment:
It would be ideal to do this with a unit test that includes just a few lines of PDB-format text. Instead of adding a whole new PDB file to the resources. Would that be doable?

Reply:
Done